src/main/resources/com/rackspace/jenkins_nodepool/NodePoolSlave/configure-entries.jelly

```diff
@@ -4,39 +4,72 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-
     <f:section title="NodePool Agent Properties">
         NodePool agents/slaves are ephemeral and last for the duration of a single build unless the Hold flag is set.
 
         <f:entry title="${%Hold Node?}" field="held">
-            <f:checkbox readonly="${descriptor.holdReadOnly}"/>
+            <f:checkbox/>
+        </f:entry>
+
+        <f:entry title="${%User Requesting Hold:}">
+            <j:choose>
+                <j:when test="${it.holdUser==null}">
+                    <j:choose>
+                        <j:when test="${it.node.userId.equals('anonymous')}">
+                            ${%anonymous user}
+                            <input type="hidden" name="holdUser" value="${%anonymous user}" />
+                        </j:when>
+                        <j:otherwise>
+                            <a href="${app.rootUrl}${user.url}">${user.displayName}</a>
+                            <input type="hidden" name="holdUser" value="${user.displayName}" />
+                        </j:otherwise>
+                    </j:choose>
+                </j:when>
+                <j:otherwise>
+                    ${it.holdUser}
```

Comment:
Link here?

When testing, the first time I viewed the configure page (to enable hold), my username was shown linked. When I re-opened the configure page to release the link, my username was plain text, presumably because of this line?

`${app.rootUrl}/user/${it.holdUser}` should do it.

Reply:
Will remove link - was toying with the idea of linking to the user in all cases/views.

Reply:
Removed link.